ca/server.go

```diff
@@ -521,19 +524,21 @@ func (s *Server) isRunning() bool {
 // anything to update the root CA material
 func (s *Server) UpdateRootCA(ctx context.Context, cluster *api.Cluster) {
 	s.mu.Lock()
+	defer s.mu.Unlock()
```

Comment:
It doesn't seem super long-lived, as we're not generating any new creds, so this seems relatively short.  I can create a separate lock, though if we like, just for updating the security config maybe.